perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_entityless_attribute_def_def_nonAuthzPwdResetMailTemplate_namespace.java

```diff
@@ -0,0 +1,42 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.EntitylessAttributesModuleAbstract;
+
+/**
+ * non authorized password reset mail template attribute
+ *
+ * @author Daniel Fecko dano9500@gmail.com
+ */
+public class urn_perun_entityless_attribute_def_def_nonAuthzPwdResetMailTemplate_namespace extends EntitylessAttributesModuleAbstract {
+
+	@Override
+	public void checkAttributeValue(PerunSessionImpl perunSession, String key, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException {
+		if (attribute.getValue() == null) return;
+		if (!(attribute.getValue() instanceof String)) {
+			throw new WrongAttributeValueException(attribute, key, "value must be of type String");
+		}
+		String value = attribute.valueAsString();
+		if (!value.contains("{link}")) {
+			throw new WrongAttributeValueException(attribute, key, "value doesn't contain contain tag {link}");
+		}
+	}
+
+	@Override
+	public AttributeDefinition getAttributeDefinition() {
+		AttributeDefinition aD = new AttributeDefinition();
+		aD.setNamespace(AttributesManager.NS_ENTITYLESS_ATTR_DEF);
+		aD.setFriendlyName("nonAuthzPwdResetMailTemplate_namespace");
```

Comment:
There is no ``_namespace`` in the friendlyName of attribute, since specific namespace is used on attribute creation. Attribute without namespace doesn't exist. So you can delete whole ``getAttributeDefinition()`` since in this contex it doesn't make sense (its similar to login-namespace attribute).